templates/ethstore.html

```diff
@@ -0,0 +1,524 @@
+          <ol>
+            <li>Consensus rewards: Rewards received by validators for proposing and attesting in accordance with the consensus rules of the Ethereum protocol</li>
+            <li>Transaction fees: Fees received by validators for processing transactions on the Ethereum network</li>
+          </ol>
+          <p>Total staking rewards equals the sum of consensus rewards and transaction fees</p>
+        </div>
+
+        <button type="button" class="collapsible">How can I reproduce the ETH.STORE calculation on my own?</button>
+        <div class="content">
+          <div class="table-responsive">
+            <table class="table">
+              <thead>
+                <tr>
+                  <th>Variable</th>
+                  <th>Description</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>Start epoch (SE):</td>
+                  <td>First epoch after 1200 UTC every day</td>
+                </tr>
+                <tr>
+                  <td>End epoch (EE):</td>
+                  <td>SE + 225</td>
+                </tr>
+                <tr>
+                  <td>Active validators (VS):</td>
+                  <td>Validators that have been active from start SE to end EE [SE,EE)</td>
+                </tr>
+                <tr>
+                  <td>Sum rewards (RS):</td>
+                  <td>Sum of all rewards (consensus, tx) in VS that have been generated in epochs [SE,EE)</td>
+                </tr>
+                <tr>
+                  <td>Sum effective balances (EB):</td>
+                  <td>Sum of all effective balances at epoch SE of all validators in VS</td>
+                </tr>
+                <tr>
+                  <td>ETH.STORE:</td>
+                  <td>(RS/EB)*365</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+
+          <br />
+          <p>
+            Calculations for <strong><span aria-ethereum-date="{{ .Data.YesterdayTs }}" aria-ethereum-date-format="dd MMMM yyyy"></span> 12:00:23 UTC - <span aria-ethereum-date="{{ addI64 .Data.YesterdayTs 86400 }}" aria-ethereum-date-format="dd MMMM yyyy"></span> 12:00:22 UTC:</strong>
+          </p>
+
+          <div class="table-responsive">
+            <table class="table">
+              <thead>
+                <tr>
+                  <th>Variable</th>
+                  <th>Description</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>Start epoch (SE):</td>
+                  <td>Epoch {{ .Data.StartEpoch }}</td>
+                </tr>
+                <tr>
+                  <td>End epoch (EE):</td>
+                  <td>Epoch {{ addUI64 .Data.StartEpoch 225 }}</td>
+                </tr>
+                <tr>
+                  <td>Active validators (VS):</td>
+                  <td>in timeframe [{{ .Data.StartEpoch }},{{ addUI64 .Data.StartEpoch 225 }})</td>
```

Comment:
Three times a "[" is combined with a ")". This should be changed to "[" and "]" for all cases, should it not?

Again, this matches the behavior over on https://staking.ethermine.org/statistics but I think this should be changed on both sides.

Reply:
I believe this is written in mathematical interval notation, as in [ is including that number while ) is excluding that number

Reply:
TIL!

Thanks for your explanation => Resolved.